newscorp-ghfb/newskit
# chore(PPDSC-2079): update axe-core and fix any errors

PPDSC-2079

**What**

Our `axe-core` and `cypress-axe` versions were not up to date, so I've updated them and cleared all new a11y errors.

**I have done:**

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.




**How to test:**
Run `yarn e2e:build` then `yarn e2e:comps` and check if all tests pass (but that should be handled by circleci either way).
